ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/BlockHeaderBuilder.java

```diff
@@ -87,7 +87,7 @@ public static BlockHeaderBuilder fromHeader(final BlockHeader header) {
   }
 
   public static BlockHeaderBuilder fromBuilder(final BlockHeaderBuilder fromBuilder) {
-    BlockHeaderBuilder toBuilder =
+    final BlockHeaderBuilder toBuilder =
```

Comment:
why is this called toBuilder when the method is called fromBuilder? (I realise you did not change this...)

Reply:
I think it makes sense. This is the builder that we are copying to. fromBuilder is the builder that we are copying from.